app/components/view-section-table-row/macro.test.js

```diff
@@ -66,4 +68,108 @@ describe('view-section-table-row', () => {
         done();
       });
   });
+
+  it('should render a horizontal row if layout is not provided', (done) => {
+    const context = {
+      params: {
+        questionId: 'some-question-id',
+        innerComponent: '<p>Some inner component</p>',
+      },
+    };
+
+    const dummyApp = createTestHarness(macroWrapper, context);
+    request(dummyApp)
+      .get('/')
+      .then((res) => {
+        const $ = cheerio.load(res.text);
+
+        expect($('[data-test-id="view-section-table-row-horizontal"]').length).toEqual(1);
+
+        done();
+      });
+  });
+
+  it('should render a vertical row if layout is set to "horizontal"', (done) => {
```

Comment:
think you mean `vertical`

Reply:
or do you mean `horizantal` the test is checking for horizantal

Reply:
I did mean horizontal.